fe/fe-core/src/main/java/com/starrocks/connector/iceberg/cost/IcebergStatisticProvider.java

```diff
@@ -96,13 +96,13 @@ private IcebergFileStats generateIcebergFileStats(IcebergTable icebergTable, Sca
                 if (dataFile.recordCount() == 0) {
                     continue;
                 }
+                if (files.contains(dataFile.path().toString())) {
+                    continue;
+                }
+                files.add(dataFile.path().toString());
```

Comment:
If recordCount() is always 0, IcebergFileStats may be null

Reply:
done